components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,177 @@
+
+import { ReactComponent as Alert } from "../images/exclamation.svg";
+import { ReactComponent as Success } from "../images/tick.svg";
+import { ReactComponent as Tick } from "../images/check.svg";
+import { ReactComponent as Cross } from "../images/x.svg";
+
+export default function License() {
+    const { license, setLicense } = useContext(LicenseContext);
+
+    useEffect(() => {
+        if (isGitpodIo()) {
+            return; // temporarily disable to avoid hight CPU on the DB
+        }
+        (async () => {
+            const data = await getGitpodService().server.adminGetLicense();
+            setLicense(data);
+        })();
+    }, []);
+
+    const featureList = license?.enabledFeatures;
+    const features = license?.features;
+
+    // if user seats is 0, it means that there is user limit in the license
+    const userLimit = license?.seats == 0 ? "Unlimited" : license?.seats;
+
+    const [licenseLevel, paid, msg, tick] = getSubscriptionLevel(
+        license?.plan || "",
+        license?.userCount || 0,
+        license?.seats || 0,
+        license?.fallbackAllowed || false,
+    );
+
+    return (
+        <div>
+            <PageWithSubMenu
+                subMenu={adminMenu}
+                title="License"
+                subtitle="License associated with your Gitpod Installation"
+            >
+                <div className="flex flex-row space-x-4">
+                    <Card className="bg-gray-800 dark:bg-white text-white dark:text-gray-400">
+                        <p className="text-white dark:text-black font-bold pt-4 text-sm"> {licenseLevel}</p>
+                        <p className="dark:text-gray-500">{paid}</p>
+                        <p className="text-gray-400 pt-4 pb-2">Available features:</p>
+                        {features &&
+                            features.map((feat: string) => (
+                                <div className="flex">
+                                    {featureList?.includes(feat) ? (
+                                        <Tick className="h-4" />
+                                    ) : (
+                                        <Cross className="h-4 w-4 px-1" />
+                                    )}
+                                    {capitalizeInitials(feat)}
+                                </div>
+                            ))}
+                    </Card>
+                    <Card className="bg-gray-200 dark:bg-gray-900 text-gray-600 dark:text-gray-600">
+                        <div className="text-gray-600 dark:text-gray-200 text-sm py-4 flex-row flex items-center">
+                            <div>{msg}</div>
+                            <div className="px-4">{getLicenseValidIcon(tick)}</div>
+                        </div>
+                        <p className="dark:text-gray-500">Registered Users</p>
+                        <span className="dark:text-gray-300 pt-1 text-lg">{license?.userCount || 0}</span>
```

Comment:
**question**: Could we align and fix the action button on bottom right area of the card as described in the design specs? Using flex layout could help or simply using some bottom margin here if that sounds safe. Your call. 🏓 

| BEFORE | AFTER |
|-|-|
| ![button-before](https://user-images.githubusercontent.com/120486/164090297-c57a626e-91bf-4d65-a457-9759c3cb43c1.png) | ![button-after](https://user-images.githubusercontent.com/120486/164090312-dcd997c9-ba38-4c07-8340-f32280dc9517.png) |

Reply:
**suggestion**: Since we're here could we also move the checkmark to that the right margin matches the right margin of the button at the bottom of this card? 😇

Reply:
I have made the changes!